src/applications/static-pages/createMyVALoginWidget.js

```diff
@@ -1,12 +1,15 @@
 import { isLoggedIn } from '../../platform/user/selectors';
+import isBrandConsolidationEnabled from '../../platform/brand-consolidation/feature-flag';
+
+const dashboardPath = isBrandConsolidationEnabled() ? '/my-va' : '/dashboard';
```

Comment:
For this, you can just import the Dashboard manifest. The `receiveContentProps` will have already been executed, so the `rootUrl` will be an accessible property. This is sorta too abstracted, but it kept the changes we needed to make for brand-consolidation to minimum, and saves us some teardown when the feature flag is ready to be removed.